### ISSUE TITLE
picklist - listbox violations

### ISSUE THREAD

**Issue by geromegrignon** [openng-org/optimus-ui#275]
### Describe the bug

The p-picklist component has an accessibility violation when running the IBM Equal Accessibility Checker: "Interactive component with ARIA role 'tree' does not have a programmatically associated name"

### Reason for not contributing a PR

- [ ] Lack of time
- [ ] Unsure how to implement the fix/feature
- [ ] Difficulty understanding the codebase
- [ ] Other

### Reproducer

https://stackblitz-starters-bwt9ucm2.stackblitz.io

### Environment

Stackblitz

### Angular version

20.0.4

### PrimeNG version

v20

### Steps to reproduce the behavior

1. Install the [IBM Equal Accessibility Checker](https://chromewebstore.google.com/detail/ibm-equal-access-accessib/lkcagbfjnkomcinoddgooolagloogehp)
2. Open the stackblitz
3. Run the accessibility checker
4. Filter on violations only and note the error: 'Interactive component with ARIA role 'tree' does not have a programmatically associated name'

### Expected behavior

No accessibility violations are present for the p-picklist and p-listbox component

> Migrated from `primefaces/primeng#18941` — originally by `@srujana10499` on 2025-09-26

**Comment by geromegrignon** [comment]
*[Originally posted by `@github-actions[bot]` on 2026-03-26T00:31:06.000Z]*

This issue has been automatically marked as stale due to a lack of activity. **If this issue is still relevant to you, please leave a comment** so we can keep it open. We apologize for not being able to prioritize it sooner. If you have any new information or questions, please share them in your comment!
